lib/Tool/Requirements.php

```diff
@@ -107,19 +107,19 @@ public static function checkMysql(ConnectionInterface $db)
         $largePrefix = $db->fetchRow("SHOW GLOBAL VARIABLES LIKE 'innodb\_large\_prefix';");
         $checks[] = new Check([
             'name' => 'innodb_large_prefix = ON ',
-            'state' => ($largePrefix && $largePrefix['Value'] != 'ON') ? Check::STATE_ERROR : Check::STATE_OK
+            'state' => ($largePrefix && !in_arrayi($largePrefix['Value'], ['on', '1', 1])) ? Check::STATE_ERROR : Check::STATE_OK
         ]);
 
         $fileFormat = $db->fetchRow("SHOW GLOBAL VARIABLES LIKE 'innodb\_file\_format';");
         $checks[] = new Check([
             'name' => 'innodb_file_format = Barracuda',
-            'state' => ($fileFormat && $fileFormat['Value'] != 'Barracuda') ? Check::STATE_ERROR : Check::STATE_OK
+            'state' => ($fileFormat && !in_arrayi('barracuda', $fileFormat['Value'])) ? Check::STATE_ERROR : Check::STATE_OK
```

Comment:
`$fileFormat['Value']` is not an array here, is it?
Do you mean `strtolower($fileFormat['Value']) !== 'barracuda'`
Or you could do `!in_arrayi('barracuda', [$fileFormat['Value']])`

Reply:
@BlackbitNeueMedien Thanks, yes you are true, i keeped arrayi because used everywhere in this method, but if it is not an array it will raise an error or notice i think as arrayi use array_map whose parameter must be an array.
So it is better to replace it like you suggest by: `strtolower($fileFormat['Value']) != 'barracuda'` (i keeped only `!=` because it was a non strict non egality in the original code, but strict non equality is ok too!)

Reply:
@BlackbitNeueMedien @brusch I just made a new commit to fix this array problem and add same principe of strtolower to utf8mb4 check in case of.
Should be ok now and ready to merge : )